### ISSUE TITLE
[REGTEST] Default PerfectMoney Account is listed as USD, but all strings are in EUR

### ISSUE THREAD

**Issue by julianknutsen** [bisq-network/bisq#3693]
### Description

When starting a regtest environment for the first time, a default PerfectMoney account is created that is listed as USD. But, on the Create Offer screen, all values are listed in EUR.

#### Version

v1.2.4
06f0d6b191d46d2ec877ab0707c8a4815b0e00d3

### Steps to reproduce

1. Start new regtest cluster and set up mediator/refund agent
2. Create Buy BTC offer on Alice
3. See PerfectMoney (USD) account selected, but values in EUR

### Expected behaviour

Values should be in USD

### Actual behaviour

Values are in EUR

### Screenshots

![perfect_money_eur](https://user-images.githubusercontent.com/8082291/69661426-395f1600-1037-11ea-9965-1a6da7e7841e.JPG)

#### Device or machine

Ubuntu 19.10

#### Additional info

Toggling between the Eth Dummy and Perfect Money has no effect
Creating a new PerfectMoney account with USD causes both the new AND old to properly show values in USD.

There must be some sort of state updated on account creation that the dummy account does not trigger. But, once another account is created it works.

**Comment by chimp1984** [comment]
It might be a bug from the Dummy accounts (created in dev mode at startup) and currency derived from the locale. 
Does this happen also if you create a new account? Questionable if we should keep those dummy accounts around...

**Comment by julianknutsen** [comment]
No, all new accounts do the right thing. And creating a new USD Perfect Money account "fixes" the dummy account.

It is nice to have one-less-step to set up a regtest cluster, but if the code paths are different it is probably best to just remove them.

**Comment by chimp1984** [comment]
Actually AliPayAccount is used in setupDevDummyPaymentAccounts not PerfectMoney. If you got that maybe it was still in the data directory presets? I remember to switch to AliPayAccount because PerfectMoney had multi currencies and there was some issue with locales and initialisation.

**Comment by julianknutsen** [comment]
Ya, I just recopied master's dao-setup.zip to reset everything. Is there a different way I should be initializing the state for these tests?

**Comment by chimp1984** [comment]
That is the easiest way. We could just clean up that account (delete it an create a new one) and then upload that zip.

**Comment by ripcurlx** [comment]
I recognized this bug as well with the default setup file. I'll prepare a new setup zip for the default setup.
